grin/src/Reducer/PrimOps.hs

```diff
@@ -139,19 +139,19 @@ evalPrimOp name params args = case name of
     _ -> error $ "invalid arguments:" ++ show params ++ " " ++ show args ++ " for " ++ unpackName name
 
   string_cons = case args of
-    [RT_Lit (LInt64 a), RT_Lit (LString b)] -> string $ Text.cons (chr (fromIntegral a)) b
+    [RT_Lit (LInt 64 a), RT_Lit (LString b)] -> string $ Text.cons (chr (fromIntegral a)) b
     _ -> error $ "invalid arguments: " ++ show params ++ " " ++ show args ++ " for " ++ unpackName name
 
   int_str = case args of
-    [RT_Lit (LInt64 a)] -> string $ fromString $ show a
+    [RT_Lit (LInt 64 a)] -> string $ fromString $ show a
```

Comment:
Is it intended to use a specific width in each of these functions?

Reply:
So far, I have converted primops from `T_Int64` to `T_Int 64`, but the end goal is to generalize them (part of checkbox three, I should probably specify that). Note that operations will only be defined for numbers of the same width, i.e. `add :: i8 -> i7 -> i9` should be invalid (same as how LLVM does it).

Reply:
I see, thanks!